tests/src/test/scala/com/softwaremill/sttp/HttpURLConnectionHttpTest.scala

```diff
@@ -0,0 +1,9 @@
+package com.softwaremill.sttp
+
+import com.softwaremill.sttp.testing.{ConvertToFuture, HttpTest}
+
+class HttpURLConnectionHttpTest extends HttpTest[Id] {
```

Comment:
shouldn't this now go into the tests inside the `core` module? Probably the whole `tests` module wouldn't be needed at all?

Reply:
I think that would be better, but it is more of an integration test than a unit test. Should it just go with the existing unit tests or should the integration scope be added?

Reply:
Well almost all of the tests are integration tests, there's a rather small number of "unit" tests under the restricted definition. I think their number is small enough not to divide them :)

Reply:
Done